src/Authentication/Controllers/SystemUserController.cs

```diff
@@ -242,4 +242,26 @@ public async Task<ActionResult<Paginated<SystemUser>>> GetAllSystemUsersByVendor
 
         return null;
     }
+
+    /// <summary>
+    /// Creates a new SystemUser.
+    /// </summary>
+    /// <returns>SystemUser response model</returns>    
+    [Authorize(Policy = AuthzConstants.POLICY_ACCESS_MANAGEMENT_WRITE)]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(SystemUser), StatusCodes.Status201Created)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [HttpPost("{party}/bff")]
```

Comment:
Why prefix. This is potential a API that can be called by other than bff. No need to call it BFF if we are not doing something special for bFF

Reply:
ok, route is now party/create instead of party/bff